src/main/scala/wdl4s/cwl/CwlEncoders.scala

```diff
@@ -0,0 +1,22 @@
+package wdl4s.cwl
+
+import eu.timepit.refined.api.Refined
+import io.circe.{Encoder, Json}
+import wdl4s.cwl.CwlType.CwlType
+import wdl4s.cwl.CwlVersion.CwlVersion
+import io.circe.syntax.EncoderOps
+
+/**
+  * wdl4s
+  * Created by oruebenacker on 06.07.17.
+  */
+object CwlEncoders {
+  implicit val cwlVersionEncoder: Encoder[CwlVersion] =
+    (cwlVersion: CwlVersion) => Json.fromString(cwlVersion.toString)
+  implicit val ecmaScriptExpressionEncoder: Encoder[ECMAScriptExpression] =
+    (a: ECMAScriptExpression) => Json.fromString(a.toString)
+  implicit val cwlTypeEncoder: Encoder[CwlType] =
+    (cwlType: CwlType) => Json.fromString(cwlType.toString)
+  implicit def refinedEncoder[V,P](implicit valueEncoder: Encoder[V]): Encoder[Refined[V, P]] =
```

Comment:
I don't think this boilerplate is necessary as long as `import io.circe.refined._`is done.